scytale.yaml

```diff
@@ -418,3 +418,18 @@ service:
   #       # (Optional) defaults to empty struct
   #       # queryOptions:
   #       #   useCache: true
+# traceProvider is the configuration for otel
+traceProvider:
```

Comment:
Since this is required, we should add it to https://github.com/xmidt-org/scytale/blob/main/deploy/packaging/scytale_spruce.yaml as well

Reply:
added tracing configuration in scytale_spruce.yaml